README.md

```diff
@@ -1,115 +1 @@
-# venn [![Build Status](https://travis-ci.org/bitoiu/venn.png)](https://travis-ci.org/bitoiu/venn) [![NPM version](https://badge.fury.io/js/venn.png)](http://badge.fury.io/js/venn)
```

Comment:
> If both projects are maintained we should rename this file instead of deleting it. I guess it makes sense, otherwise we would delete the whole thing and separate two projects.

💯  agree